msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1112,14 +1113,31 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
 |`requirements.scopes`| Array of enums | 1 | | Identifies the scopes in which the add-in can run and defines the Microsoft 365 applications in which the extension can run. For example, `mail` (Outlook). <br>Supported value: `mail` |
 |`requirements.formFactors`| Array of enums | | | Identifies the form factors that support the add-in. <br>Supported values: `mobile`, `desktop`|
-|`alternateIcons`| Object | | | Specifies the main icons that are used to represent the add-in on older versions of Office. This property is **required** if the Office add-in is to be installable in Office on Mac, perpetual Office licenses, and Microsoft 365 subscription versions of Office on Windows earlier than 2304 (Build 16320.00000).|
+|`alternateIcons`| Object | | | Specifies the main icons that are used to represent the add-in on older versions of Office. This property is **required** if the Office Add-in is to be installable in Office on Mac, perpetual Office licenses, and Microsoft 365 subscription versions of Office on Windows earlier than 2304 (Build 16320.00000).|
 |`alternateIcons.icon`| Object | | ✔️ | Specifies properties of the image file used to represent the add-in. |
 |`alternateIcons.icon.size`| Number enum | | ✔️ | This property is reserved for future use. The value must be 64.|
 |`alternateIcons.icon.url`| String | 2048 characters | ✔️ | Specifies the full, absolute URL of the image file that is used to represent the add-in. Icon image must be 64 x 64 pixels and use one of the following file formats: GIF, JPG, PNG, EXIF, BMP, TIFF.|
 |`alternateIcons.highResolutionIcon`| Object | | ✔️ | Specifies properties of the image file used to represent the add-in on high DPI screens. |
 |`alternateIcons.highResolutionIcon.size`| Number enum | | ✔️ | This property is reserved for future use. The value must be 64 (not 128). |
 |`alternateIcons.highResolutionIcon.url`| String | 2048 characters | ✔️ | Specifies the full, absolute URL of the image file that is used to represent the add-in on high DPI screens. Icon image must be 128 x 128 pixels and use one of the following file formats: GIF, JPG, PNG, EXIF, BMP, TIFF.|
 
+### extensions.contextMenus
+
+A context menu is a shortcut menu that appears when you right-click (or select and hold) in an open Office document.
+
+|Name | Type | Maximum size | Required | Description |
+|---|---|---|---|---|
+|`menus`| Array | | ✔️ | Configures the context menus. |
+|`menus.entryPoint`| String enum | | ✔️ | Use `text` or `cell` here for Office context menu. Use `text` if the context menu should open when a user right-clicks (or selects and holds) on the selected text. Use `cell` if the context menu should open when the user right-clicks (or selects and holds) on a cell in an Excel spreadsheet. |
+|`menus.controls`| Array | | ✔️ | The same as controls in [extensions.ribbons.tabs.groups.controls](#extensionsribbons). The control type should be `button`. |
```

Comment:
Hey @ElizabethSamuel-MSFT,
First of all, apologies for the recurring reviews as we couldn't finish it in one-go due to other priorities. I will make arrangements to review and merge this PR asap. 

In this line: 1132, though I really like the idea of linking to a similar property (I have also proposed it once), we finally decided to explicitly list all the properties with the same data so the developers can cross-check and make sure the properties, default values, max size are specific and tailored to the specific feature they are defining. Request you to list the sub-properties of `menus.controls` for additional clarity for developers.

Reply:
@v-ramyas5 I've added all the extensions.ribbons.tabs.groups.controls entries here, updating them to menus.controls.*. However, I'll need some help to understand which should be removed/updated. Can you and @Rick-Kirkham help with that?

Reply:
@ElizabethSamuel-MSFT , I did this in the form of a very long suggestion comment.